### ISSUE TITLE
Missing "replaceable" declaration

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#595]
**Reported by stefanv on 29 Jul 2011 16:21 UTC**
In Modelica.Media.Air.MoistAir (for example) contains:

```
redeclare replaceable model extends BaseProperties(
    ...
    redeclare final constant Boolean standardOrderComponents = true)
```

BaseProperties is inherited from Modelica.Media.Interfaces.PartialCondensingGas, which in turn inherits it from Modelica.Media.Interfaces.PartialMixtureMedium, which in turn inherits it from Modelica.Mediua.Interfaces.PartialMedium, where it is declared:

```
replaceable partial model BaseProperties
   ...
   parameter Boolean standardOrderComponents = true;
   ...
```

Notice that standardOrderComponents is<ins>not</ins> declared **replaceable**. Thus the redeclaration of it is not allowed.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/595

**Comment by modelica-trac-importer** [comment]
**Comment by hubertus on 28 May 2012 22:16 UTC**
fixed in 7ac6785a7007f93aafa2f4c432dfa3485c9e94e7

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 29 May 2012 06:01 UTC**
To me it is not clear when this can be set to be fixed as it's not included in trunk yet. So leaving as open until it is merged.

**Comment by modelica-trac-importer** [comment]
**Comment by hubertus on 10 Jun 2012 23:14 UTC**
Fixed in a251c81439d91417bda21fba4ef0f945a3854365

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 25 Jul 2012 14:48 UTC**
